ViewLayer/DisplayDashboardUI/I_DisplayDashboardUI.h

```diff
@@ -1,53 +0,0 @@
-#pragma once
```

Comment:
Why did we decide to have this file removed again? I'm not sure that's the best idea anymore.